great_expectations/datasource/data_connector/inferred_asset_azure_data_connector.py

```diff
@@ -0,0 +1,132 @@
+import logging
+import os
+from typing import List, Optional
+
+from great_expectations.core.batch import BatchDefinition
+from great_expectations.core.batch_spec import PathBatchSpec, AzureBatchSpec
+from great_expectations.exceptions.exceptions import ParserError
+
+from great_expectations.datasource.data_connector import (
+    InferredAssetFilePathDataConnector,
+)
+from great_expectations.datasource.data_connector.util import list_azure_keys
+from great_expectations.execution_engine import ExecutionEngine
+
+logger = logging.getLogger(__name__)
```

Comment:
Please add the `try/except` clause we have for cloud provider connection objects (see S3 or ConfiguredAssetAzure imports)